static/styles/subscriptions.scss

```diff
@@ -658,22 +658,6 @@ form#add_new_subscription {
     position: relative;
 }
 
-.stream-row .sub-info-box .top-bar > div {
-    display: inline-block;
-    vertical-align: top;
-}
-
-.stream-row .sub-info-box .top-bar .stream-name,
-.stream-row .sub-info-box .bottom-bar .description {
-    white-space: nowrap;
-    overflow: hidden;
-    text-overflow: ellipsis;
-}
```

Comment:
This CSS block seems to have been accidentally just deleted?

Please be really careful with your process to avoid mistakes like this; they can be very hard to catch, and it's like 20x the work to later find and fix weird behavior caused by accidentally deleting CSS in a refactoring.

Reply:
Really sorry for this, un-did this.